content/content/hackathon_2020_savethedate.md

```diff
@@ -0,0 +1,53 @@
+---
```

Comment:
I want to create a "single" page for the "save the date" and I was trying to point to it from the navbar when people cliclk "hackathon" for example but I can't seem to make this happen.
Locally the address would be: http://localhost:1313/content/hackathon_2020_savethedate/
But I have no idea how to do this once it is served in production.

Reply:
Hi @Remi-Gau! Thanks a lot for those updates, they look great! I've tried playing up with the website a little bit to see if I could help here... The "fresh" theme uses a single-page layout so this might explain why you can't have hackathon pointing to a separate "save-the-date" page. *But*, we should be able to have the "hackathon" link in the top nav bar brings us to the subsection on the hackathon (in the same page). Unfortunately, I can't get the links in the original theme ([demoed here](https://themes.gohugo.io//theme/hugo-fresh/)) to work on my machine (not on Firefox and not on Chrome), does it work for you when you click "Pricing" at https://themes.gohugo.io//theme/hugo-fresh/?

Reply:
Hey. Yeah I gave up on creating a separate page with this theme accepting that it is not made for that.
I also tried to find a way to link from the navbar to the section in the main (and only) page but I could not make it work... Will wait till my patience tank is a bit more full before I get back to it.

Reply:
Hi @Remi-Gau: in https://github.com/Remi-Gau/ossig/pull/1, I've updated the "hackathon" link to point to the save-the-date section of the long page. Let me know if that's better.

Reply:
Works like a charm. Merci beaucoup